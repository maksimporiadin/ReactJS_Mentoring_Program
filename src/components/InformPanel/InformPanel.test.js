import 'jsdom-global/register';
import React from 'react';

import { InformPanel } from './index';

import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

configure({adapter: new Adapter()});

describe('InformPanel', () => {
    let wrapper;

    it('is renders', () => {
        wrapper = shallow(<InformPanel  />);
        expect(wrapper).toHaveLength(1);
    });
});