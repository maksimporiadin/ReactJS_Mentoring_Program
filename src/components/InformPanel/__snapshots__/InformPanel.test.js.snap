// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InformPanel is renders 1`] = `
<div
  className="informPanel"
/>
`;
